package.json

```diff
@@ -93,7 +91,9 @@
 		"micromatch": "^4.0.8",
 		"trim-newlines": "^3.0.1",
 		"parse-path": "^5.0.0",
-		"http-cache-semantics": "^4.1.1"
+		"http-cache-semantics": "^4.1.1",
+		"send": "^0.19.0",
```

Comment:
We may not be needing `send` or `axios` as dependencies since I can't find any logic in the CLI that uses them.

Reply:
Done